infra/label_evaluator_step_functions/lambdas/llm_review.py

```
@@ -0,0 +1,2284 @@
+                        "Marking %d issues as NEEDS_REVIEW.", len(chunk_metadata)
+                    )
+                    for i, meta in enumerate(chunk_metadata):
+                        ctx = issues_with_context[i] if i < len(issues_with_context) else {}
+                        decisions["NEEDS_REVIEW"] += 1
+                        reviewed_issues.append({
+                            "image_id": meta["image_id"],
+                            "receipt_id": meta["receipt_id"],
+                            "issue": meta["issue"],
+                            "llm_review": {
+                                "decision": "NEEDS_REVIEW",
+                                "reasoning": f"LLM call failed: {e}",
```

Ruff (0.14.10)

Undefined name `e`

(F821)